tools/topology/platform/common/ssp.m4

```diff
@@ -27,16 +27,19 @@ define(`SSP_CONFIG',
 $6
 )
 
-dnl SSP_CONFIG_DATA(type, idx, valid bits, mclk_id)
+dnl SSP_CONFIG_DATA(type, idx, valid bits, quirks, mclk_id)
 dnl mclk_id is optional
 define(`SSP_CONFIG_DATA',
 `SectionVendorTuples."'N_DAI_CONFIG($1$2)`_tuples" {'
 `	tokens "sof_ssp_tokens"'
 `	tuples."word" {'
 `		SOF_TKN_INTEL_SSP_SAMPLE_BITS'	STR($3)
 `	}'
+`	tuples."word" {'
+`		SOF_TKN_INTEL_SSP_QUIRKS'	ifelse($4, `', "0", STR($4))
+`	}'
```

Comment:
@fredoh9 this can go in the same tuples."word" {}

Reply:
got it